### PR TITLE
General mutect2 wdl

### PR DESCRIPTION
**Pull request by takutosato**
`scripts/mutect2_wdl/mutect2-sge.wdl` takes the following parameters and runs GATK4 Mutect2 in SGE.

* gatk4 jar
* tumor bam (and its index, tumor sample name)
* normal bam (likewise)
* reference
* pon (optional)
* scatter count

`scripts/mutect2_wdl/mutect2-dream-eval-sge.wdl` takes a gatk4 jar and scatter count and runs the full dream evaluation.

An example input json for each case is also included.

### CODE REVIEW

**Comment by takutosato** [comment]
@davidbenjamin please review

File: scripts/mutect2_wdl/mutect2-sge.wdl [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    SCATTER_COUNT=${scatterCount}
+  >>>
+
+  output {
+    Array[File] intervalFiles = glob("intervalFileDir/temp_*/scattered.interval_list")
+  }
+}
+
+workflow Mutect2 {
+	File gatk4_jar
```
Comment (davidbenjamin) [inline scripts/mutect2_wdl/mutect2-sge.wdl]: extra indent

**Comment by davidbenjamin** [comment]
@takutosato Very minor stuff.  You can squash and merge without sending it back.

File: scripts/mutect2_wdl/mutect2-dream-eval-sge.wdl [hunk 2]
```diff
@@ -0,0 +1,191 @@
+# how to run
+# 1) start a cromwell server:
+# java -Dconfig.file=sge_pc_application_ruchi_cc.conf -jar $cromwell server
```
Comment (takutosato) [inline scripts/mutect2_wdl/mutect2-dream-eval-sge.wdl]: I added the config file `sge.conf` to the repo. The identical copy of `sge.conf` is in `/humgen/gsa-hpprojects/dev/tsato/wdl/sge_pc_application_ruchi.conf`.

File: scripts/mutect2_wdl/mutect2-sge.wdl (hunk 1 above)
Comment (takutosato) [inline scripts/mutect2_wdl/mutect2-sge.wdl]: done

**Pull request merged**